apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx

```diff
@@ -9,16 +10,14 @@ export const HeadlineBanner = () => {
           Track our progress as an ENS Service Provider
         </p>
       </div>
-      <a
+      <Link
```

Comment:
This link is supposed to get more attention. That's why it should display with an underline such as https://storybook.namekit.io/?path=/story/ui-link--underline-medium